src/profile/profile.controller.ts

```diff
@@ -45,58 +47,46 @@ export class ProfileController {
     return ZaLaResponse.Ok(userProfile, 'Profile created', 201);
   }
 
-  @Get('/:profileId')
-  @IdCheck('profileId')
+  @Get()
   @ApiOperation({ summary: 'Get a profile' })
-  async getOne(
-    @Param('profileId', new ParseUUIDPipe()) profileId: string,
-  ): Promise<Ok<Profile>> {
-    const profile = await this.profileService.getone(profileId);
+  async getOne(@Req() req: Request): Promise<Ok<Profile>> {
+    const profile = await this.profileService.getone(req.profileId);
     return ZaLaResponse.Ok(profile, 'Ok', 200);
   }
 
-  @Patch(':profileId')
-  @IdCheck('profileId')
-  @UseGuards(AuthorizationGuard)
+  @Patch()
   @ApiOperation({ summary: 'Update an existing profile' })
   async updateProfile(
-    @Param('profileId', ParseUUIDPipe) profileId: string,
+    @Req() req: Request,
```

Comment:
What if an Admin wants to update the profile?

Reply:
I asked @PhilemonBrain on friday and he said it's only the user that can updaate his profile for now

Reply:
Okay